src/test/kotlin/io/github/mojira/arisa/modules/LanguageModuleTest.kt

```diff
@@ -183,6 +183,114 @@ class LanguageModuleTest : StringSpec({
         result.shouldBeRight(ModuleResponse)
     }
 
+    "should pass the combined text to the API" {
+        val module = LanguageModule()
+        val request = Request(
+            NOW,
+            A_SECOND_AGO,
+            "Summary.",
+            "Description.",
+            "not private",
+            "private",
+            { it shouldBe "Summary. Description."; emptyMap<String, Double>().right() },
+            { Unit.right() },
+            { Unit.right() }
+        )
+
+        module(request)
+    }
```

Comment:
You forgot the `result.shouldBeRight(ModuleResponse)` part in these

Reply:
Intended actually, since I only want to test the text parsed to the API in these tests

Reply:
I would test both in this case

Reply:
Then you need to make sure the function is actually called in some other way. E.g. by defining a variable at the top, setting it to the text in the function, and check the variable at the end.

Reply:
That's a nice point!